### DISCUSSION TITLE
Add option to disable Whats new in the VSCode Extension

### DISCUSSION THREAD

**Discussion by dougg0k** [SeaQL/FireDBG.for.Rust#28] [Ideas]
It's currently forcefully appearing everytime there is a new version (changelog). 

An option to disable that through config would be cool, and a button on the same UI.

**Comment by tyt2y3** [comment]
Sure, I get you.

**Reply by dougg0k** [reply]
Still appearing a window everytime, mostly with rustc changes in changelog. 

Does not seem relevant.

**Reply by tyt2y3** [reply]
sorry, it does, but it happens only once per update

**Reply by dougg0k** [reply]
Yeah, will just keep it disabled by default and enable per project.
